src/main/java/pl/javastart/streamsexercise/PaymentService.java

```diff
@@ -19,70 +22,170 @@ class PaymentService {
     Znajdź i zwróć płatności posortowane po dacie malejąco
      */
     List<Payment> findPaymentsSortedByDateDesc() {
-        throw new RuntimeException("Not implemented");
+        List<Payment> paymentList = paymentRepository.findAll();
+        paymentList.sort(new PaymentDateComparator());
+        return paymentList;
```

Comment:
To też jest ok, ale chciałbym, żebyś w każdej metodzie użyła struieni, czyli `paymentRepository.findAll().stream()...`